packages/angular-sdk-components/src/lib/_components/field/location/location.component.ts

```diff
@@ -0,0 +1,356 @@
+import { ChangeDetectorRef, Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { GoogleMapsModule } from '@angular/google-maps';
+import { GoogleMapsLoaderService } from '../../../_services/google-maps-loader.service';
+import { CommonModule } from '@angular/common';
+import { FormControl, FormGroup, ReactiveFormsModule } from '@angular/forms';
+import { debounceTime, from, interval, of, switchMap } from 'rxjs';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import { MatAutocompleteModule } from '@angular/material/autocomplete';
+import { MatProgressSpinnerModule } from '@angular/material/progress-spinner';
+import { MatIconModule } from '@angular/material/icon';
+import { MatButtonModule } from '@angular/material/button';
+import { AngularPConnectData, AngularPConnectService } from '../../../_bridge/angular-pconnect';
+import { Utils } from '../../../_helpers/utils';
+import { PConnFieldProps } from '../../../_types/PConnProps.interface';
+import { handleEvent } from '../../../_helpers/event-util';
```

Comment:
lets reorder the imports 
@angular imports (core, common, material, google) -> third party libraries (rxjs) -> local components -> services -> utils -> types